From fe14fd1a18c73dc618863f8fe5c5cc4e46671853 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Fri, 9 May 2014 16:19:14 +0800
Subject: [PATCH 29/30] ENGR00304389 Frame drop is almost 90% with WFD display

Only statics the frame dropped as the front frame been replaced

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 libs/gui/BufferQueue.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/libs/gui/BufferQueue.cpp b/libs/gui/BufferQueue.cpp
index 4fe2035..8b7f808 100644
--- a/libs/gui/BufferQueue.cpp
+++ b/libs/gui/BufferQueue.cpp
@@ -590,10 +590,11 @@ status_t BufferQueue::queueBuffer(int buf,
                 }
                 // and we record the new buffer in the queued list
                 *front = item;
+                mFrameLost ++;
             } else {
                 mQueue.push_back(item);
                 listener = mConsumerListener;
-                mFrameLost ++;
+
             }
         }
 
-- 
1.8.0

